ios/push-notifications.md

```diff
@@ -57,38 +58,44 @@ Use the following steps to connect add push notification backend services to you
+	If you encounter an error message that begins `[!] Failed to connect to GitHub to update the CocoaPods/Specs...`, and your internet connectivity is working, you may need to [update openssl and Ruby](https://stackoverflow.com/questions/38993527/cocoapods-failed-to-connect-to-github-to-update-the-cocoapods-specs-specs-repo/48962041#48962041).
 
-2. Classes that call Amazon Pinpoint APIs must use the following import statements:
+1. Classes that call Amazon Pinpoint APIs must use the following import statements:
 
 	```
 	import AWSCore
 	import AWSPinpoint
+    import AWSMobileClient
 	```
 
-3. Create an Amazon Pinpoint client by using the following code into the
-         `didFinishLaunchwithOptions` method of your app's `AppDelegate.swift`. This
-         will also register your device token with Amazon Pinpoint.
+1. To receive push notifications with Amazon Pinpoint, you'll instantiate a Pinpoint instance and register your device token with Amazon Pinpoint. We recommend you do this during app startup, so your users can begin receiving notifications as early as possible.
+
+    **Note**: If you have already integrated `Analytics`, you can skip this step.
 
-         Note: If you have already integrated `Analytics`, you can skip this step.
+    Edit the `application(_:didFinishLaunchingWithOptions:)` method of your app's `AppDelegate.swift` by adding a `pinpoint` instance property, and initializing the Pinpoint client as shown below:
 
-         ```swift
-             /** start code copy **/
-             var pinpoint: AWSPinpoint?
-             /** end code copy **/
+    ```swift
+    class AppDelegate: UIResponder, UIApplicationDelegate {
 
+       /** start code copy **/
+       var pinpoint: AWSPinpoint?
+       /** end code copy **/
 
-             func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions:
-                 [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+       func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
 
-                 /** start code copy **/
-                 pinpoint = AWSPinpoint(configuration:
-                         AWSPinpointConfiguration.defaultPinpointConfiguration(launchOptions: launchOptions))
-                 /** end code copy **/
+            // Other didFinishLaunching code...
 
-                 return true
-             }
-             ```
+            /** start code copy **/
+            // Initialize Pinpoint
+            let pinpointConfiguration = AWSPinpointConfiguration.defaultPinpointConfiguration(launchOptions: launchOptions)
```

Comment:
Are we using `awsconfiguration.json` here? If so we should show a snippet of the structure and required keys, usually this is under a "Manual Configuration" section. This allows customers to use with an existing appId if they didn't setup using the CLI.

Reply:
Done. I copied the format of the manual configuration section in https://docs.aws.amazon.com/aws-mobile/latest/developerguide/how-to-integrate-an-existing-bucket.html